internal/engine/engine_state.go

```diff
@@ -45,9 +47,19 @@ func newManifestState(manifest model.Manifest) *manifestState {
 		lastBuild:          BuildStateClean,
 		manifest:           manifest,
 		pendingFileChanges: make(map[string]bool),
+		pod:                unknownPod,
 	}
 }
 
+type Pod struct {
+	Name      string
+	StartedAt time.Time
+	Status    string
+}
+
+// manifestState.Pod will be set to this if we don't know anything about its pod
+var unknownPod = Pod{Name: "no pod yet found"}
```

Comment:
i'm surprised you didn't use the zero value for this?

Reply:
Yeah, I need to get more comfortable with zero values. I'm still getting used to not having `null` or `Option`